execution_engine_testing/cargo_casper/tests/integration_tests.rs

```diff
@@ -73,6 +73,8 @@ fn run_tool_and_resulting_tests() {
         .arg(nightly_version.trim())
         .arg("cargo")
         .arg("test")
+        .arg("--")
+        .arg("--nocapture")
```

Comment:
Not sure how spammy that output is; do we always want that?

Reply:
Nope, thanks.